src/device/mock.ts

```diff
@@ -49,9 +50,7 @@ export class MockDeviceConnection
 
   async initialize(): Promise<void> {}
 
-  dispose() {
-    this.removeAllListeners();
```

Comment:
We can't do this anymore as EventTarget doesn't have such API.

All listener add/removes are correctly paired except that in host.ts and a test, neither of which need to remove their listeners.